test/acceptance/framework/helpers/helpers.go

```diff
@@ -57,7 +57,7 @@ func WaitForAllPodsToBeReady(t *testing.T, client kubernetes.Interface, namespac
 // Sets up a goroutine that will wait for interrupt signals
 // and call cleanup function when it catches it.
 func SetupInterruptHandler(cleanup func()) {
-	c := make(chan os.Signal)
+	c := make(chan os.Signal, 1)
```

Comment:
why are we changing this to a buffered channel?

Reply:
This was flagged by the linter.
"The os/signal package will not block when sending a signal, which means that the channel passed to signal.Notify should always be buffered." - https://github.com/golang/lint/issues/175

Reply:
👍 thanks!